pkg/cloudprovider/gce/gce.go

```diff
@@ -479,6 +479,17 @@ func (gce *GCECloud) UpdateTCPLoadBalancer(name, region string, hosts []string)
 			return err
 		}
 	}
+
+	// Try to verify that the correct number of nodes are now in the target pool.
```

Comment:
Can you comment on why we're testing this particular edge case, why we don't try to fix it, and what has to happen for it to get fixed?